wordpress-mobile/WordPress-Editor-iOS
# Fix iphone x issues

This PR solves layout issues on the iPhoneX for the legacy editor and the hybrid editor.

How to test:
 - Launch the demo app using the iPhoneX.
 - Scroll around on the demo content in visual mode and HTML mode.
 - Check if not overlap happens on the corners and bottom of the iPhone X
 - Check that making keyboard visible and invisible doesn't brake anything
 - Test the legacy editor for the same things.
 - Test on the older devices to see if nothing is broken.
